quelpa-use-package.el

```diff
@@ -37,17 +37,35 @@
 
 ;;; Code:
 
-(defvar quelpa-use-package-inhibit-loading-quelpa nil
+(defgroup quelpa-use-package nil
+  "Quelpa handler for `use-package'."
+  :prefix "quelpa-use-package"
+  :group 'use-package)
+
+(defcustom quelpa-use-package-inhibit-loading-quelpa nil
   "If non-nil, `quelpa-use-package' will do its best to avoid
```

Comment:
The first line of a docstring should be a complete sentence.  If this one's is being updated, it might as well be fixed.